frame/support/src/lib.rs

```diff
@@ -123,7 +123,7 @@ impl TypeId for PalletId {
 #[macro_export]
 #[cfg(feature = "std")]
 macro_rules! bounded_vec {
-	($ ($values:expr),* ) => {
+	($ ($values:expr),* $(,)?) => {
```

Comment:
Weird, I thought `$($foo:expr),*` is already enough to allow for a trailing comma?

Reply:
No `$($foo:expr),*` means that you allow expressions that are separated by a comma. If there is no more expression coming, no extra comma is allowed.

Reply:
Ah right, I was thinking of `$($foo:expr,)*`, but that one means that a comma MUST follow an expression.